.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: native-metrics CI
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm ci
+    - name: Run build
+      run: npm run build
+    - name: Run Unit
+      run: npm run unit
+    - name: Run Integration
+      run: npm run integration
+    - name: Upload to S3
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.S3_BUCKET_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_BUCKET_SECRET_ACCESS_KEY }}
+      run: npm run upload
```

Comment:
I think you need an extra line at the end of the file

Reply:
think it usually shows a red circle when there's no line. so might be fine.

Reply:
hmm, my editor doesn't like it (RED STUFF), and it seems other workflow files don't have an extra line 🤷

Reply:
no worries then :)